### PR TITLE
[UI]: Removing nav-bar and hamburger menu

### LINKED ISSUE #217: [UI] Remove duplicate navigation bars
STR:

1. Navigate to http://localhost:5080/shell

screen shot 2018-12-24 at 3 48 50 pm

Actual: There are 2 navigation bars on the left

Expected: There is 1 navigation bar on the left

### PR DESCRIPTION
**Pull request by vishakhanihore**
Github Bug/Issue: Fixes #217

### CODE REVIEW

**Comment by vishakhanihore** [comment]
@djmitche @helfi92 Please have a look

File: ui/src/views/Shell/index.jsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import ShellConsole from '../../components/Shell';
-import Dashboard from '../../components/Dashboard';
```
Comment (helfi92) [inline ui/src/views/Shell/index.jsx]: What's nice about using the Dashboard component is that it sets, the logo icon and page title in the browser tab:

Screen Shot 2019-11-18 at 9 05 53 AM

We should probablty still use the Dashboard component but expose a new prop `disableAppbar` that hides the app bar if the prop is provided:

```diff
diff --git a/ui/src/components/Dashboard/index.jsx b/ui/src/components/Dashboard/index.jsx
index b438f9a10..fb96f0b75 100644
--- a/ui/src/components/Dashboard/index.jsx
+++ b/ui/src/components/Dashboard/index.jsx
@@ -165,6 +165,7 @@ export default class Dashboard extends Component {
     helpView: null,
     docs: false,
     disableTitleFormatting: false,
+    disableAppbar: false,
   };
 
   static propTypes = {
@@ -199,6 +200,10 @@ export default class Dashboard extends Component {
      * If true, the title will not be formatted to uppercase.
      */
     disableTitleFormatting: bool,
+    /**
+     * If true, the app bar will not be rendered.
+     */
+    disableAppbar: bool,
   };
 
   static getDerivedStateFromError(error) {
```

**Comment by vishakhanihore** [comment]
Cool, I will changes.

**Comment by helfi92** [comment]
@vishakhanihore, How is it going? :-)

**Comment by vishakhanihore** [comment]
Updating tonight 😅

File: ui/src/views/Shell/index.jsx (hunk 1 above)
Comment (vishakhanihore) [reply ui/src/views/Shell/index.jsx]: @helfi92 I made the change and introduced the new prop, How do I make the use of the prop to hide the menu ?

Comment (helfi92) [reply ui/src/views/Shell/index.jsx]: If `disableAppbar` is `true`, then do not render the app bar (i.e., `<Appbar .. />`). Else, render the app bar.

**Comment by helfi92** [comment]
@vishakhanihore Are you still working on this?

**Pull request closed without merging**